src/generator/ModuleGenerators/TS.ts

```diff
@@ -126,11 +126,7 @@ export const renderTypeScriptDeclarationForDocumentModels = (
       //
       //
 
-      import type { Settings } from ${
-        settings.data.output.directory === 'default'
-          ? `'../generator/Settings'`
-          : `'nexus-prisma/dist-cjs/generator/Settings'`
-      }
+      import type { Settings } from 'nexus-prisma/dist-cjs/generator/Settings/index'
```

Comment:
I don't know why I must add `index` at the end, or it will break.

Reply:
Are you overriding `--moduleResolution` somewhere? Resolution to a file named `index` is a node thing

Reply:
I think it is because ESM requires file paths. It does not support Node resolution algorithm. But Maybe this is something else, because if this were ESM related I would expect that the file extension is required too.